pkgs/applications/graphics/emulsion-palette/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, meson
+, ninja
+, vala
+, wrapGAppsHook4
+, libadwaita
+, json-glib
+, libgee
+, pkg-config
+, gtk3
+, desktop-file-utils
+}:
+
+stdenv.mkDerivation rec {
+  pname = "emulsion-palette";
+  version = "3.3.9";
+
+  src = fetchFromGitHub {
+    owner = "lainsce";
+    repo = "emulsion";
+    rev = version;
+    sha256 = "sha256-xG7yZKbbNao/pzFhdTMof/lw9K12NKZi47YRaEd65ok=";
+  };
+
+  nativeBuildInputs = [ meson ninja pkg-config vala wrapGAppsHook4 ];
+
+  buildInputs = [
+    desktop-file-utils
+    gtk3 # We're only using it for the gtk-update-icon-cache utility.
+    json-glib
+    libadwaita
+    libgee
+  ];
+
+  meta = with lib; {
+    mainProgram = "io.github.lainsce.Emulsion";
```

Comment:
I think we should symlink the binary to just `emulsion`/`emulsion-palette` as well, seeing how this is a bit lengthy.

Reply:
Done, seeing this is also how some applications like [Foliate](https://github.com/NixOS/nixpkgs/blob/c8ee69fb01f979e8980e1517c06eac8236776e85/pkgs/applications/office/foliate/default.nix) are packaged due to lengthy executable names. Though, I named the symlink `emulsion-palette` since there is [another package](https://github.com/NixOS/nixpkgs/blob/634141959076a8ab69ca2cca0f266852256d79ee/pkgs/applications/graphics/emulsion/default.nix) with the same binary.